class1/README.md

```diff
@@ -1 +1,131 @@
 #Class 1
+
+##Goals: 
```

Comment:
I don't think we need to include this here, it was more for us to think about as we plan the classes.